benches/src/lib.rs

```diff
@@ -418,6 +453,12 @@ impl TryFrom<VmBench> for VmBenchPrepared {
             db.deploy_contract_with_id(&slots, &contract, &id)?;
         }
 
+        if let Some(BlobCode { code, id }) = blob {
+            db.storage_as_mut::<BlobData>()
+                .insert(&id, &code.0)
+                .unwrap();
+        }
```

Comment:
Any reason why we unwrap here instead of just returning an error?

```suggestion
        if let Some(BlobCode { code, id }) = blob {
            db.storage_as_mut::<BlobData>()
                .insert(&id, &code.0)?;
        }
```